app.rb

```diff
@@ -64,11 +69,13 @@ def check_selection(response)
     when '5'
       book = Book.create_book
       @books << book
-      puts 'Book created successfully' if @books.include?(book)
+      puts "\n\nBook created successfully\n\n" if @books.include?(book)
+      # json_book = JSON.generate({ title: book.title, author: book.author })
+      # File.write("books.json", "#{json_book},\n", mode: "a")
```

Comment:
- [ ] Please remove commented code. Getting rid of commented code helps declutter your codebase and helps you have clean code 👍